charts/external-es-proxy/values.yaml

```diff
@@ -9,6 +9,10 @@ images:
     repository: openresty/openresty
     pullPolicy: IfNotPresent
     tag: latest
+  awsproxy:
+    repository: abutaha/aws-es-proxy
```

Comment:
Saw the license which is permissive. Should we package and use an astronomer/ap-aws-es-proxy  so in the field we don't hit docker hub issues and also are able to verify the provenance of the image? @danielhoherd

Reply:
yes, absolutely. we should vendor this into ap-vendor.

Reply:
will be adding it to ap-vendor